vectorstores/chroma/chroma.go

```diff
@@ -0,0 +1,196 @@
+package chroma
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	chromago "github.com/amikos-tech/chroma-go"
+	"github.com/amikos-tech/chroma-go/openai"
+	"github.com/google/uuid"
+	"github.com/tmc/langchaingo/embeddings"
+	"github.com/tmc/langchaingo/internal/util"
+	"github.com/tmc/langchaingo/schema"
+	"github.com/tmc/langchaingo/vectorstores"
+)
+
+var (
+	ErrInvalidScoreThreshold    = errors.New("score threshold must be between 0 and 1")
+	ErrUnexpectedResponseLength = errors.New("unexpected length of response")
+	ErrNewClient                = errors.New("error creating collection")
+	ErrAddDocument              = errors.New("error adding document")
+	ErrRemoveCollection         = errors.New("error resetting collection")
+	ErrUnsupportedOptions       = errors.New("unsupported options")
+)
+
+// Store is a wrapper around the chromaGo API and client.
+type Store struct {
+	client           *chromago.Client
+	collection       *chromago.Collection
+	distanceFunction chromago.DistanceFunction
+	chromaURL        string
+	openaiAPIKey     string
```

Comment:
what is this apikey for in this Store?

Reply:
The "Open API Key" is the key used to interact with the OpenAI LLM.